### ISSUE TITLE
Plain return statement fails

### ISSUE THREAD

**Issue by hkaiser** [STEllAR-GROUP/phylanx#815]
This code throws a font-end error:
```py
from phylanx import Phylanx

@Phylanx
def test():
    return 

test()
```
Here is the actual error message:
```
Traceback (most recent call last):
  File "test.py", line 9, in <module>
    @Phylanx
  File "...\phylanx\ast\transducer.py", line 92, in Phylanx
    return __PhylanxDecorator(__phylanx_arg)
  File "...\phylanx\ast\transducer.py", line 43, in __init__
    kwargs)
  File "...\phylanx\ast\physl.py", line 351, in __init__
    self.__src__ = self.generate_physl(self.ir)
  File "...\phylanx\ast\physl.py", line 371, in generate_physl
    result = [self.generate_physl(i) for i in ir[1]]
  File "...\phylanx\ast\physl.py", line 371, in <listcomp>
    result = [self.generate_physl(i) for i in ir[1]]
  File "...\phylanx\ast\physl.py", line 371, in generate_physl
    result = [self.generate_physl(i) for i in ir[1]]
  File "...\phylanx\ast\physl.py", line 371, in <listcomp>
    result = [self.generate_physl(i) for i in ir[1]]
  File "...\phylanx\ast\physl.py", line 371, in generate_physl
    result = [self.generate_physl(i) for i in ir[1]]
  File "...\phylanx\ast\physl.py", line 371, in <listcomp>
    result = [self.generate_physl(i) for i in ir[1]]
  File "...\phylanx\ast\physl.py", line 369, in generate_physl
    if len(ir) == 2 and isinstance(ir[0], str) and isinstance(
TypeError: object of type 'NoneType' has no len()
```

**Comment by hkaiser** [comment]
@rtohid What's the status of this? Any news?

**Comment by rtohid** [comment]
Sorry, I had missed this. I have a fix for the front-end issue (#897). 
Meanwhile, if the return value is None, Phylanx functions return `nil` instead of `None` (please see #896):

```py
from phylanx import Phylanx

@Phylanx(debug=1)
def ret():
    return

print(ret())
```
returns:
```
define(
    ret,
    lambda(nil)
)
nil
```
